src/adapter.ts

```diff
@@ -273,28 +273,10 @@ export class MongooseAdapter {
   loadPolicyLine(line: policyLine, model: Model) {
     let lineText = line.p_type;
 
-    if (line.v0) {
-      lineText += ', ' + line.v0;
-    }
-
-    if (line.v1) {
-      lineText += ', ' + line.v1;
-    }
-
-    if (line.v2) {
-      lineText += ', ' + line.v2;
-    }
-
-    if (line.v3) {
-      lineText += ', ' + line.v3;
-    }
-
-    if (line.v4) {
-      lineText += ', ' + line.v4;
-    }
-
-    if (line.v5) {
-      lineText += ', ' + line.v5;
+    for (const word of [line.v0, line.v1, line.v2, line.v3, line.v4, line.v5]) {
+      if (word !== undefined) {
+        lineText += `, ` + word;
```

Comment:
I kinda hate +-strings. Wouldn't
```typescript
lineText = `${lineText}, ${word}`
```
look cleaner? Not sure if the node version is high enough for format-strings, but just a thought. Also, since there is no else-statements, I would just remove the curly brackets and make it into one liner.  Just my personal taste however. i.e.

```typescript
if (word !== undefined) lineText = `${lineText}, ${word}`
```